projects/libyaml/build.sh

```diff
@@ -18,9 +18,17 @@
 ./configure
 make "-j$(nproc)"
 
-$CXX $CXXFLAGS -std=c++11 -Iinclude \
-    $SRC/libyaml_fuzzer.cc -o $OUT/libyaml_fuzzer \
-    $LIB_FUZZING_ENGINE src/.libs/libyaml.a
+for fuzzer in $(find $SRC/fuzz -name '*_fuzzer.cc'); do
```

Comment:
Nit: you can replace the `$(find ...)` expression with a regular glob `$SRC/fuzz/*_fuzzer.cc`.